app/views/layouts/_head.html.erb

```diff
@@ -13,22 +13,21 @@
   <%= csp_meta_tag %>
   <%= content_for(:head) %>
 
-  <script type="text/javascript">
-      var _paq = window._paq || [];
-      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
-      _paq.push(['trackPageView']);
-      _paq.push(['enableLinkTracking']);
-      (function () {
-          var u = "https://analytics.libraries.psu.edu/matomo/";
-          _paq.push(['setTrackerUrl', u + 'matomo.php']);
-          _paq.push(['setSiteId', <%= Settings.matomo_id %>]);
-          var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
-          g.type = 'text/javascript';
-          g.async = true;
-          g.defer = true;
-          g.src = u + 'matomo.js';
-          s.parentNode.insertBefore(g, s);
-      })();
+  <!-- Matomo -->
+  <script>
+    var _paq = window._paq = window._paq || [];
+    /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+      var u="https://analytics.libraries.psu.edu/matomo/";
+      _paq.push(['setTrackerUrl', u+'matomo.php']);
+      _paq.push(['setSiteId', '13']);
```

Comment:
we shouldn't hardcode the site_id here. I think we were doing it right before, with `Settings.matomo_id` I think we just need to update the matomo_id in the prod deployment to be 11.